hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -56,6 +56,11 @@
           "by Container Balancer.")
   private long maxSizeToMove = 10 * OzoneConsts.GB;
 
+  @Config(key = "idle.iterations", type = ConfigType.INT,
```

Comment:
Based on current implementation, I would suggest changing the "idle.iterations" to "max.iterations".  As Lokesh said, we can do it in a seperate PR later to revisit all these configurations and namings.

Reply:
yea, we could revisit it later